go/kbfs/libkbfs/prefetcher.go

```diff
@@ -1349,6 +1423,29 @@ func (p *blockPrefetcher) run(
 	isShuttingDown := false
 	var shuttingDownCh <-chan interface{}
 	first := true
+	appState := keybase1.MobileAppState_FOREGROUND
+	netState := keybase1.MobileNetworkState_NONE
+
+	// Subscribe to settings updates while waiting for the network to
+	// change.
+	subMan := p.config.SubscriptionManager()
+	var subCh chan struct{}
+	if subMan != nil {
+		subCh = make(chan struct{}, 10)
```

Comment:
why 10?

Reply:
Just thought it would be enough to avoid blocking the subscription manager.  Settings don't change very much.

But it's more clear to just have it be of size 1, and to skip sending on it if it's already full, so I just changed it to do that.